### DISCUSSION TITLE
Apple Magic keyboard Home/End (Fn+LEFT/RIGHT) keys not work?

### DISCUSSION THREAD

**Discussion by sentinel3** [termux/termux-app#2588] [General]
Hello I am quite new to termux, I recently get a Samsung Galaxy Tab S7+ Tablet (Android 11) and use a physical keyboard (Apple Magic keyboard) for some occasional work. 
In termux, all the normal letter, number keys work well, even the "ctrl + ..." functions. the problem is the `Fn` key functions. since the keyboard does not physically have the `Home`, `End`, and `Delete`, it uses: `Fn+Left`, `Fn+Right`, `Fn+Backescape` instead. the above 3 keys works well in other apps such as GoogleDocs on the tablet, only have problem in Termux.

In termux, if I type `Fn+left` expect a `Home` button pressed, it will flash with the `menu` pop up window (below) very fast, then nothing happens. 
![Screenshot_20220214-001028_Termux](https://user-images.githubusercontent.com/48333411/153804698-3148a1f9-7390-4edd-9a72-7ae2abe4a956.jpg)

I checked the key event with another app, where the 3 above key show events:
```
Press buttons to see the keycodes:
(Home: FN+left)
KeyDown     : action=0 code=119 repeat=0 meta=2097160 scanCode=464 flags=8 label='0' chars='null' number='0'
KeyUp       : action=1 code=122 repeat=0 meta=2097160 scanCode=102 flags=8 label='0' chars='null' number='0'
KeyUp       : action=1 code=119 repeat=0 meta=2097152 scanCode=464 flags=8 label='0' chars='null' number='0'

(End: FN+right)
KeyDown     : action=0 code=119 repeat=0 meta=2097160 scanCode=464 flags=8 label='0' chars='null' number='0'
KeyDown     : action=0 code=123 repeat=0 meta=2097160 scanCode=107 flags=8 label='0' chars='null' number='0'
KeyUp       : action=1 code=123 repeat=0 meta=2097160 scanCode=107 flags=8 label='0' chars='null' number='0'
KeyUp       : action=1 code=119 repeat=0 meta=2097152 scanCode=464 flags=8 label='0' chars='null' number='0'

(Delete: FN+backspace)
KeyDown     : action=0 code=119 repeat=0 meta=2097160 scanCode=464 flags=8 label='0' chars='null' number='0'
KeyDown     : action=0 code=112 repeat=0 meta=2097160 scanCode=111 flags=8 label='0' chars='null' number='0'
KeyUp       : action=1 code=112 repeat=0 meta=2097160 scanCode=111 flags=8 label='0' chars='null' number='0'
KeyUp       : action=1 code=119 repeat=0 meta=2097152 scanCode=464 flags=8 label='0' chars='null' number='0'
```

following the [help](https://wiki.termux.com/wiki/Terminal_Settings), I tried to add customized keys in the `~/.termux/termux.properties` file, but also not work:
```
HOME=FN+LEFT 
END=FN+RIGHT 
DEL=FN+BKSP
```
any suggestion?  thx!

**Comment by ChiefMikeK** [comment]
When reporting an issue please post as per the instructions (last item on the `menu`) please, fully read to the bottom, edit re-dact any personal info and post as a collapsed item.  
- [How to add a collapsible section in markdown.](https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab )
- Q: could you try temp switching launguage of the phone 
- check current lang setting of termux by `echo $LANG` should be default:
en_US.UTF-8
- Too change it `set $LANG=en_GB.UTF-8` or any other language. then open a new session tab
